MonoGame.Framework/MacOS/MacGamePlatform.cs

```diff
@@ -380,6 +380,13 @@ public override void ResetElapsedTime()
         {
             _gameWindow.ResetElapsedTime();
         }
+		
+		public override void Present()
+        {
+            var device = Game.GraphicsDevice;
+            if (device != null)
+                device._graphicsMetrics = new GraphicsMetrics();

```

Comment:
Than this issue resolve without my PR, because OpenTKGamePlatform already have Present.